pkg/kubelet/kuberuntime/kuberuntime_sandbox_test.go

```diff
@@ -167,6 +167,7 @@ func TestCreatePodSandbox_RuntimeClass(t *testing.T) {
 				assert.NoError(t, err)
 				assert.Contains(t, fakeRuntime.Called, "RunPodSandbox")
 				assert.Equal(t, test.expectedHandler, fakeRuntime.Sandboxes[id].RuntimeHandler)
+				assert.Equal(t, test.expectedHandler, fakeRuntime.Sandboxes[id].Annotations["kubernetes.io/runtimehandler"])
```

Comment:
and since we are not going to force the annotations map into existence we need to do the assert for the annotation only if expected.. to avoid the nil ptr